graviti/schema/catalog_to_schema.py

```diff
@@ -0,0 +1,324 @@
+#!/usr/bin/env python3
+#
+# Copyright 2022 Graviti. Licensed under MIT License.
+#
+
+"""Functions converting TensorBay Catalog to Graviti Schema."""
+
+from functools import reduce
+from pathlib import Path
+from typing import Any, Dict, List, Tuple, Union
+
+from tensorbay.dataset import Notes, RemoteData
+from tensorbay.label import (
+    Catalog,
+    ClassificationSubcatalog,
+    InstanceMaskSubcatalog,
+    Keypoints2DSubcatalog,
+    Label,
+    LabeledKeypoints2D,
+    PanopticMaskSubcatalog,
+)
+from tensorbay.label.basic import SubcatalogBase, _LabelBase
+from tensorbay.label.supports import AttributesMixin, CategoriesMixin, MaskCategoriesMixin
+
+_JSON_TYPE_TO_PORTEX_TYPE = {
+    "array": "array",
+    "boolean": "boolean",
+    "integer": "int32",
+    "number": "float32",
+    "string": "string",
+}
+
+
+_PYTHON_TYPE_TO_PORTEX_TYPE = {
+    int: "int32",
+    float: "float32",
+}
+
+_LABEL_PROCESSORS: Dict[str, "_ConverterBase"] = {}
+
+
+_FILE_TYPES_AND_SUFFIX = {
```

Comment:
Set filetypes as key?

Will it be better if just remove `_FILE_TYPES_AND_SUFFIX` and put them into the generate function?
```python
for extension in (".bmp", ".jpg", ".jpeg", ".png", ".gif"):
    _SUFFIX_AND_FILE_FIELDS[extension] = {"name": "image", "type": "file.Image"}
for extension in (".pcd", ".ply", ".las", ".laz"):
     _SUFFIX_AND_FILE_FIELDS[extension] = {"name": "point_cloud", "type": "file.PointCloud"}
```